OvernightPrintsSpecBindings/TestBase/Config/XmlConfigReader.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Xml;
+
+namespace OvernightPrintsSpecBindings.TestBase.Config
+{
+	class XmlConfigReader
+	{
+		private readonly string path = "C:\\Users\\dalekseev\\Config.xml";

```

Comment:
It's better to use Environment.SpecialFolder. It will allow to run with any user, not only dalekseev